openmrs-central/prod.my.cnf

```diff
@@ -0,0 +1,23 @@
+[mysqld]
```

Comment:
Which mysql instance is this for? Recall there is several MySQL instances

Reply:
This is for openmrs db.

Reply:
ok